src/enzo/global_data.h

```diff
@@ -967,17 +967,27 @@ EXTERN char LocalPath[MAX_LINE_LENGTH];
 EXTERN char GlobalPath[MAX_LINE_LENGTH];
 #endif
 
-#ifdef USE_PYTHON
 EXTERN int NumberOfPythonCalls;
 EXTERN int NumberOfPythonTopGridCalls;
 EXTERN int NumberOfPythonSubcycleCalls;
+#ifdef USE_PYTHON
 EXTERN PyObject *grid_dictionary;
 EXTERN PyObject *old_grid_dictionary;
 EXTERN PyObject *hierarchy_information;
 EXTERN PyObject *yt_parameter_file;
 EXTERN PyObject *conversion_factors;
 EXTERN PyObject *my_processor;
 #endif
+
+#ifdef USE_LIBYT
+EXTERN void *param_libyt;
+EXTERN void *param_yt;
+EXTERN int libyt_field_lookup[FieldUndefined];
+#endif
+EXTERN int NumberOfLibytCalls;
+EXTERN int NumberOfLibytTopGridCalls;
+EXTERN int NumberOfLibytSubcycleCalls;
```

Comment:
Not sure why these are outside the `#ifdef USE_LIBYT` clause if they are associated with libyt. This is more of an organizational difference than technical. If it breaks compilation to have these variables conditionally defined, obviously leave it be.

Reply:
It might, but I think we can investigate.